src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/controllers/PersonControllerTest.kt

```diff
@@ -28,11 +30,21 @@ internal class PersonControllerTest(
     }
 
     it("responds with a 200 OK status") {
+      val person = Person(id, "Billy", "Bob")
+      Mockito.`when`(getPersonService.execute(id)).thenReturn(person)
       val result = mockMvc.perform(get("/persons/$id")).andReturn()
 
       result.response.status.shouldBe(200)
     }
 
+    it("responds with a 404 NOT FOUND status") {
+      val id_that_does_not_exist = 777
+
+      mockMvc.perform(get("/persons/$id_that_does_not_exist"))
+        .andDo(print())
+        .andExpect(status().isNotFound)
```

Comment:
Could we use the Kotest expectation instead e.g. `result.response.status.shouldBe(404)`?

Reply:
Ah yes, let me fix